docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
The current behavior on master (both `localhost:3000` and on the publisher test site) seems to be that clicking on usernames on annotation cards **does nothing** (on <https://hypothes.is/docs/help> in prod it links to their user page though). Looks like something's broken, to be fixed as a separate issue.

But once that's fixed - I'd like to keep this PR to _just_ the logged in user's name in the login menu, as the PR is already quite big. If it's required we can do a separate PR to also customize the other usernames and to add the arguments that's be necessary to disambiguate that?